src/components/main.tsx

```diff
@@ -68,6 +86,9 @@ const Main = () => {
                             <img src={MetaMask} alt="Connect with Metamask" />
                         </Button>
                     </InputGroupAddon>
+                    {!validAddress && <FormFeedback style={{color:'yellow'}} invalid>
+                      Uh oh! Looks like this Wallet Address is Invalid!
+                    </FormFeedback>}
```

Comment:
Additionally, please test the UI! The invalid `FormFeedback` messes up the CSS. When the invalid feedback appears, the content of the entire website moves. This should not happen. Would be great if you could fix that.